app/main.py

```diff
@@ -1,34 +1,212 @@
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int, is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
+    def __init__(self, start: tuple,
+                 end: tuple, is_drowned: bool = False) -> None:
+        self.is_drowned = is_drowned
+        self.decks = []
 
-    def get_deck(self, row, column):
-        # Find the corresponding deck in the list
-        pass
+        if start[0] == end[0] and start[1] == end[1]:
```

Comment:
The condition for identifying a single deck ship is incorrect. It should be `if start == end` instead of `if start[0] == end[0] and start[1] == end[1]`.